src/commands/classes/Contest.command.ts

```diff
@@ -1,43 +1,169 @@
 import Command from "./Command";
-import { Message } from "discord.js";
+import { Message, Channel, TextChannel } from "discord.js";
 import drive from "googleapis";
-import fs from "fs";
+import fs, { watchFile } from "fs";
 import path from 'path';
 import validUrl from 'valid-url';
 import { dataflow } from "googleapis/build/src/apis/dataflow";
+import contestData from "../../../data/contestData.json";
+import sampleFiles from "../../../templates/sampleFiles.json";
 
 export default class ContestCommand extends Command {
-  public help = "Available Sub-commands for '!Contest': \nStart \nAdd \nVote \nEnd \nHelp ";
-
+  public help = "Available Sub-commands for '!Contest': \nStart \nAdd \nVote \nEnd \nHelp "
   //initiate a contest, announcing it to channel and 
```

Comment:
All comments should follow similar formatting to the rest. Start a comment with a capital letter and a space after the double forward-slashes.